src/wasm/wasm-type.cpp

```diff
@@ -463,6 +463,8 @@ FeatureSet Type::getFeatures() const {
       if (heapType.isStruct() || heapType.isArray()) {
         return FeatureSet::ReferenceTypes | FeatureSet::GC;
       }
+    } else if (t.isRtt()) {
+      return FeatureSet::ReferenceTypes | FeatureSet::GC;
```

Comment:
Why is GC alone not enough here? AFAIK, all instructions that interact with Rtts in any way are in the GC proposal.